From 99f996769c588a52ec1b8dad6334ce3c1324ed9a Mon Sep 17 00:00:00 2001
From: Manivannan Sadhasivam <manivannan.sadhasivam@linaro.org>
Date: Fri, 7 Dec 2018 13:09:23 +0530
Subject: [PATCH 44/45] arm64: dts: hisilicon: Force 1 bit mode for Wifi on
 hikey/hikey960

Wifi only works well with default 1bit mode. Hence, remove the bus-width
property from dwmmc nodes.

Signed-off-by: Manivannan Sadhasivam <manivannan.sadhasivam@linaro.org>
---
 arch/arm64/boot/dts/hisilicon/hi3660-hikey960.dts | 1 -
 arch/arm64/boot/dts/hisilicon/hi6220-hikey.dts    | 1 -
 2 files changed, 2 deletions(-)

diff --git a/arch/arm64/boot/dts/hisilicon/hi3660-hikey960.dts b/arch/arm64/boot/dts/hisilicon/hi3660-hikey960.dts
index 43b0f64c8642..6f36cacb3eed 100644
--- a/arch/arm64/boot/dts/hisilicon/hi3660-hikey960.dts
+++ b/arch/arm64/boot/dts/hisilicon/hi3660-hikey960.dts
@@ -636,7 +636,6 @@
 };
 
 &dwmmc2 { /* WIFI */
-	bus-width = <0x4>;
 	non-removable;
 	broken-cd;
 	cap-power-off-card;
diff --git a/arch/arm64/boot/dts/hisilicon/hi6220-hikey.dts b/arch/arm64/boot/dts/hisilicon/hi6220-hikey.dts
index f4964bee6a1a..5762650a41fd 100644
--- a/arch/arm64/boot/dts/hisilicon/hi6220-hikey.dts
+++ b/arch/arm64/boot/dts/hisilicon/hi6220-hikey.dts
@@ -320,7 +320,6 @@
 		};
 
 		dwmmc_2: dwmmc2@f723f000 {
-			bus-width = <0x4>;
 			non-removable;
 			cap-power-off-card;
 			vmmc-supply = <&reg_vdd_3v3>;
-- 
2.15.1

